cfgov/jinja2/v1/external-site/index.html

```diff
@@ -28,18 +28,14 @@
 </script>
 {% endblock javascript %}
 
-{% block content_modifiers -%}
-    external-site_container
-{%- endblock %}
-
 {% block content_main %}
-    <section class='padded-container'>
+    <div class="external-site_container">
         {% if form.is_valid() %}
         <h1>Thank you for visiting consumerfinance.gov.</h1>
         <p class="lead-paragraph">
             You are leaving the CFPB web server.
             You will now access
-            <span class="external-site_text">{{ form.cleaned_data['validated_url'] }}</span>,<br>
+            <strong>{{ form.cleaned_data['validated_url'] }}</strong>,<br>
```

Comment:
This does make this slightly bolder. Is this a problem?

Reply:
I think it's fine, but it should probably be `<b>`, not `<strong>`, since it's purely for visual, not semantic, emphasis.

Reply:
Fixed!